samples/test/automlTablesPredict.v1beta1.test.js

```diff
@@ -15,74 +15,116 @@
 
 'use strict';
 
-const {assert} = require('chai');
+const {assert, expect} = require('chai');
 const {describe, it} = require('mocha');
-const {execSync} = require('child_process');
+const proxyquire = require('proxyquire');
 
 /** Tests for AutoML Tables "Prediction API" sample. */
 
-const cmdPredict = 'node automlTablesPrediction.js';
-
-// // TODO(developer): Before running the test cases, set the environment
-// variables PROJECT_ID, REGION_NAME and
-// change modelId, gcsInputUri, gcsOutputUriPrefix, bqInputUri and
-// bqOutputUriPrefix.
-//const projectId = process.env.PROJECT_ID;
-//const computeRegion = process.env.REGION_NAME;
-const modelId = 'TBL3613734080685801472';
-const filePath = './resource/predictTest.csv';
-const gcsInputUri = 'gs://automl-tables/input/test.csv';
-const gcsOutputUriPrefix = 'gs://automl-tables/output';
+const projectId = process.env.AUTOML_PROJECT_ID;
+const region = 'us-central1';
+const modelId = process.env.TABLE_MODEL_ID;
+const gcsInputUri = `gs://${projectId}-tables/predictTest.csv`;
+const gcsOutputUriPrefix = `gs://${projectId}-tables/test_outputs/`;
 const bqInputUri = 'bq://automl-tables-bg-input';
 const bqOutputUriPrefix = 'bq://automl-tables-bg-output';
 
-const exec = cmd => execSync(cmd, {encoding: 'utf8'});
+const pathStub = {};
+
+const samples = {
+  predict: {path: '../tables/predict.v1beta1.js'},
+  predictGcs: {path: '../tables/predict-gcs-source-gcs-dest.v1beta1.js'},
+  predictBq: {path: '../tables/predict-gcs-source-bq-dest.v1beta1.js'},
+};
+
+for (const sample of Object.values(samples)) {
+  sample.runSample = proxyquire(sample.path, {path: pathStub});
```

Comment:
I'm a little confused about what's happening here.  This would inject a fake implementation for a `path` module in each of the samples you're requiring.  I don't _think_ such an import exists, so it's possible this code isn't doing anything. 

What are you trying to do?

Reply:
Sorry about messy code, I have recently started learning node. There's a lot thing to learn for me.

I was using the following code as a reference.
https://github.com/googleapis/google-api-nodejs-client/blob/ef8738b2b73d9f39eac7a224e2ebce2210359141/samples/test/test.samples.drive.js#L27

I wanted to use the samples in test.js without CLI. Please guide me on this.

Reply:
Of course!  So it really depends on exactly what we want.  Do you want API calls to go all the way through to the server?  Or are you looking to mock out network calls?  If you want to prevent the calls from getting to the server, `proxyquire` is a good thing to use (this happens to be what `googleapis` does).  If you want them to go all the way through to the server, you probably don't need proxyquire at all!  

If you'd like to talk this stuff through, please feel free to grab some time with me or @bcoe (who knows more than me)

Reply:
I wouldn't use proxyquire for samples, since we've made an effort historically to make these integration tests.

I would need to better understand the goal here, a quick chat might be good 👍